packages/connectivity/src/scp-cf/identity-service.ts

```diff
@@ -4,12 +4,12 @@ import type { DestinationOptions } from './destination';
 
 /**
  * @internal
- * Make a token exchange from IAS token to XSUAA token.
- * @param options - Configuration for how to retrieve destinations from the destination service.
- * @returns Exchanged token.
+ * Make a token exchange from IAS token to XSUAA token using credentials from destination service.
+ * @param jwt - The IAS JWT to exchange.
+ * @returns Exchanged XSUAA token.
  */
 export async function exchangeToken(jwt: string): Promise<string> {
```

Comment:
[pp] I would consider renaming the function to `exchangeTokenToXsuaaToken` or something similar, as it narrows the function a little more.